LiteDB/Engine/Disk/Serializer/BufferReader.cs

```diff
@@ -328,13 +330,13 @@ public ObjectId ReadObjectId()
             }
             else
             {
-                var buffer = BufferPool.Rent(12);
+                var buffer = bufferPool.Rent(12);
```

Comment:
**Warning!** We need to to zero the loaned buffer! Otherwise the ObjectId could return unexpected results! Rel: [ArrayPool<T>.Rent(Int32) Method](https://learn.microsoft.com/en-us/dotnet/api/system.buffers.arraypool-1.rent?view=net-8.0&devlangs=csharp&f1url=%3FappId%3DDev17IDEF1%26l%3DEN-US%26k%3Dk(System.Buffers.ArrayPool%25601.Rent)%3Bk(DevLang-csharp)%26rd%3Dtrue#remarks)